Formula/binaryen.rb

```diff
@@ -18,19 +18,23 @@ class Binaryen < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "python@3.10" => :build
```

Comment:
Might not be needed given https://github.com/WebAssembly/binaryen/commit/1980e5570bce241945360abe8105101019f64bc6